src/reducers/url-state.js

```diff
@@ -256,8 +256,8 @@ const urlStateReducer: Reducer<UrlState> = (regularUrlStateReducer => (
   action: Action
 ): UrlState => {
   switch (action.type) {
-    case '@@urlenhancer/updateUrlState':
-      return action.urlState;
+    case 'URL_STATE_UPDATED_FROM_BROWSER_NAVIGATION':
+      return action.newUrlState;
```

Comment:
I was trying to disambiguate because there are 2 `UrlState`s now, the old one embedded in `State` and the newly generated `UrlState`, which has not been saved via the reducers.